Makefile

```diff
@@ -53,9 +56,15 @@ all: $(TARGET)
 
 rebuild:
 	cargo build --manifest-path libportability/Cargo.toml --features $(BACKEND)
+ifdef SYMLINK_PORTABILITY
```

Comment:
can we make this an actual make target? e.g.
```make
$(SYMLINK_DEBUG): $(LIBRARY)
  ln -sf $(CURDIR)/$(LIBRARY) $(SYMLINK_DEBUG)
```

Reply:
If we make `rebuild` depend on `$(SYMLINK)` and `$(SYMLINK)` on `$(LIBRARY)`, then won't cargo get called twice?

Reply:
Well, the `$(LIBRARY)` is just a dependency that Make would resolve (once, I assume).  Even if cargo is called the second time in a row, it would do nothing, since everything is built.

Reply:
Ok, I'll take a crack at this tomorrow.